### ISSUE TITLE
Import error using Python 3.11.0

### ISSUE THREAD

**Issue by GennadyShelukhin** [asciimoo/exrex#63]
I try using this library in Python 3.11.0 and I get an import error:
```
    import exrex
../../../env/lib/python3.11/site-packages/exrex.py:25: in <module>
    from re import sre_parse, U
E   ImportError: cannot import name 'sre_parse' from 're'
```

**Comment by sebix** [comment]
Does using `re._parser` instead work?

**Comment by GennadyShelukhin** [comment]
It doesn't work as it is now. https://github.com/python/cpython/issues/91308

**Comment by sondrelg** [comment]
@asciimoo are you still maintaining this package? Don't want to nag you, but this issue is also blocking us from upgrading to Python 3.11, so want to know whether we should look for other ways of solving the issue 🙂

**Comment by Shapedsundew9** [comment]
+1 on this. Proposed fix is good for 3.11.1.
